src/app/components/course-island/course-island.component.html

```diff
@@ -17,6 +17,14 @@ <h2 class="tui-island__title">{{ course.name }}</h2>
         <tui-tag
             [status]="course.is_registered ? 'success' : 'warning'"
             [value]="course.is_registered ? 'Registered' : 'Not Registered'"
+            class="tui-space_right-2"
```

Comment:
The convention is to use tailwind classes for spacing.
```suggestion
            class="mr-2"
```